### ISSUE TITLE
Better Mixer emotes won't render in new messages with Elixr installed

### ISSUE THREAD

**Issue by TheUnlocked** [TheUnlocked/Better-Mixer#55]
There are two parts to this issue:

1. Elixr gives plain text spans the class `me-custom-emote`, which conflicts with Better Mixer's `[class*="emote"]` blacklist on patching emotes.
2. Even if that part is fixed, the emotes are placed inside of the plain text spans rather than at the top level of message contents. This also conflicts with Better Mixer's rendering in complicated ways.

Any fix to this issue will need to be on the Elixr's side.

**Comment by TheUnlocked** [comment]
The issue has been narrowed down to just wrapping Text nodes in class-less spans, which Better Mixer now also complies with (as of b172353603de3d316f0bf4a3b183bbd87b71c4ef) and which is used by selectors (`span:not([class])`) in both extensions.

**Comment by TheUnlocked** [comment]
Just waiting on https://github.com/crowbartools/MixrElixr/pull/94

**Comment by qome** [comment]
I don't know if they updated the plugin, but Better Mixer emotes now render for me, at least the ones I put into the chat.

**Comment by TheUnlocked** [comment]
The change hasn't shown up on the Elixr repository. Because the issue is due to ordering between which extension gets to modify chat messages first, my guess is that your system has for whatever reason decided that Better Mixer (which plays nice) gets to go first.

**Comment by TheUnlocked** [comment]
https://github.com/crowbartools/MixrElixr/pull/94 has been merged. Closing.

**Comment by qome** [comment]
I'm still finding that Better Mixer only loads occassionally.

**Comment by TheUnlocked** [comment]
@qome Is this the same Elixr compatibility issue? If not, please open a new issue, and provide as specific information as possible.

**Comment by qome** [comment]
I'm assuming it's the same issue.

**Comment by TheUnlocked** [comment]
I'm not seeing the issue, even with an artificial delay to force Better Mixer to run after Elixr.

**Comment by qome** [comment]
Is there a way I can check what version of the two addons I have installed?

**Comment by TheUnlocked** [comment]
That's the same as for me. Can you describe what you're seeing in as much detail as possible?

**Comment by qome** [comment]
I guess it's a separate issue then.

**Comment by qome** [comment]
![image](https://user-images.githubusercontent.com/58382620/76708777-868af480-66c7-11ea-86a9-0d45146c3447.png)
as opposed to
![image](https://user-images.githubusercontent.com/58382620/76708786-94407a00-66c7-11ea-9372-7316635d6bfc.png)

This is after refreshing until it shows up.

**Comment by TheUnlocked** [comment]
Oh, the emote menu. That is a separate issue, but I'll take a look at it.
